db/patterns/qualified.eno

```diff
@@ -0,0 +1,12 @@
+name: Qualified
+category: site_analytics
```

Comment:
I think, it makes sense to start with that.

Looks like there is overlap with customer_interaction, but we can always split later if there is need for it.